gradlew reports throws an error with v2.0.91

Hello,
After upgrade to serenity-core v2.0.91, reports task throws an error:

> Task :reports FAILED
Generating Serenity Reports for tests to directory D:\AT\tests\target\site\serenity

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':reports'.
> Could not set unknown property 'generatedReport' for task ':reports' of type org.gradle.api.DefaultTask.

Seems like there was a typos in commit https://github.com/serenity-bdd/serenity-core/commit/bb53ffcdad57f1d077bbcd00ef9e6e56ba2c9b7e - variable 'generatedReport' is not defined 

Could you please have a look

--- a/serenity-gradle-plugin/src/main/groovy/net/serenitybdd/plugins/gradle/SerenityPlugin.groovy
+++ b/serenity-gradle-plugin/src/main/groovy/net/serenitybdd/plugins/gradle/SerenityPlugin.groovy
@@ -78,7 +78,7 @@ class SerenityPlugin implements Plugin<Project> {
                 List<String> extendedReportTypes = project.serenity.reports
                 if (extendedReportTypes) {
                     for (ExtendedReport report : ExtendedReports.named(extendedReportTypes)) {
-                        generatedReport = report.generateReportFrom(reportDirectory)
+                        def generatedReport = report.generateReportFrom(reportDirectory)
                         logger.lifecycle("  - ${report.description}: ${generatedReport.toURI()}")
                     }
                 }

